### DISCUSSION TITLE
Pi-Hole/Filebeat Ingest Parser?

### DISCUSSION THREAD

**Discussion by sjnhawk** [Security-Onion-Solutions/securityonion#9854] [Unsupported Versions]
Are there any custom ingest parsers available to parse out 'pihole.log' messages coming from filebeat?

I know that custom parsers can be placed in '/opt/so/saltstack/local/salt/elasticsearch/files/ingest/', however I am not savvy enough to write one, so I was hoping that someone in the community has one to share. Any help would be greatly appreciated.

**Comment by tomjones1977** [comment]
*Full caveat - I HAVE NOT TESTED THIS PIPELINE*

https://gist.github.com/markwalkom/7f1f20d6da7a7eb5e84843f757e2e258
